main.go

```diff
@@ -122,28 +131,52 @@ func main() {
 
 	go Listen(port)
 	fmt.Println("Listening on port " + port + " - http://localhost:" + port + "/\n")
-	buildGraph()
+
+	// TODO: Workaround: this initial call is needed only to populate RateLmitSpecs
+	buildGraph(-1)
+
 	var duration time.Duration
 	if delay != (60 * pages) {
 		duration = time.Second * time.Duration(delay)
 	} else {
-		if reflect.TypeOf(RateLimitSpecs.PollInterval).String() != "int" {
-			RateLimitSpecs.PollInterval = 60 * pages
-		}
 		duration = time.Second * time.Duration(RateLimitSpecs.PollInterval)
 	}
+
+	// TODO: refreshInterval is constant throughout the whole program. Refactor perhaps?
+	refreshInterval := int64(duration.Seconds())
```

Comment:
What would the suggested refactor be? IMO, since the delay is set dynamically at runtime via CLI flags, a refactor would give minimal performance gains over more code complexity.